khal/settings/utils.py

```diff
@@ -57,6 +57,21 @@ def is_timedelta(string):
     except ValueError:
         raise VdtValueError("Invalid timedelta: {}".format(string))
 
+def is_theme(option):
+    """checks if *theme* is a valid value
+
+    :param option: the option the user set in the config file
+    :type option: str
+    :returns: user
+    :rtype: str
+    """
+    option = option.lower()
+    if option in ['dark', 'light']:
```

Comment:
**Nitpick**: we should actually check if this is defined in `colours.py`, rather than keep a copy of the theme names here.

This is non-blocking: I'm okay with addressing this in a separate PR.